packages/auth/amplify_auth_cognito_test/test/plugin/fetch_current_device.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
*todo:* Rename the file to end in `_test.dart`

`dart test` by default looks for file names that match the pattern `*_test.dart` (see: https://pub.dev/packages/test#running-tests)

Reply:
added